extracts/v_extracts.nj_position_control_roster.sql

```diff
@@ -0,0 +1,49 @@
+USE gabby
+GO
+
+CREATE OR ALTER VIEW extracts.nj_position_control_roster AS
+
+--CTE for only titles and entities needed (including all NJ staff and CEO and CFO from KTAF)--
+WITH nj_reportable AS (
+    SELECT
+       employee_number
+      ,CASE 
+       WHEN sr.job_title = 'Chief Financial Officer' THEN '1'
+       WHEN sr.job_title = 'Chief Executive Officer' THEN '1'
+       WHEN sr.business_unit IN ('TEAM Academy Charter School','KIPP Cooper Norcross Academy') THEN 1
+       ELSE 0 END AS nj_reportable
+    FROM people.staff_roster sr
```

Comment:
use 3-part naming

Reply:
three part would be: gabby.extracts.[report name]?